hello-kubernetes/README.md

```diff
@@ -151,6 +151,45 @@ Minikube users cannot see the external IP. Instead, you can use `minikube servic
 
 Here you can see that two ports are displayed. Both the ports have been injected when Dapr was enabled for this app. Additionally, in this quickstart the HTTP Port is used for further communication with the Dapr sidecar.
 
+Next submit an order to the app
+
+<!-- STEP
+name: neworder Test
+sleep: 30
+expected_stdout_lines:
+  - ''
+env:
+  NODE_APP: "localhost:8080"
+-->
+
+```bash
+curl --request POST --data "{\"data\": { \"orderId\": \"42\" } }" --header "Content-Type:application/json" http://$NODE_APP/neworder
+```
+
+<!-- END_STEP -->
+
+Expected output:
+Empty reply from server
+
+Confirm the order was persisted by requesting it from the app
+
+<!-- STEP
+name: order Test
+expected_stdout_lines:
+  - '{"orderId":"42"}'
+env:
+  NODE_APP: "localhost:8080"
+-->
+
+```bash
+curl http://$NODE_APP/order
+```
+
+Expected output:
+```json
+{"orderId":"42"}
+```
+
```

Comment:
```suggestion
<!-- END_STEP -->

```

The missing END_STEP tag was making our autovalidation skip this step. Our parser *should* have thrown an error for this case, which means we probably have an error in our markdown automation tool: https://github.com/dapr/mechanical-markdown . I'll go ahead and fix this so we can get your PR merged, but I'll see if I can make a test for this in the upstream tool.

Reply:
I've created an issue to track the bug you exposed: https://github.com/dapr/mechanical-markdown/issues/8

Thanks!